msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -738,7 +738,7 @@ Define the properties your app uses to post a user activity feed.
 
 ## defaultInstallScope
 
-**Optional** - string
+**Optional**—string
```

Comment:
`Suggestion: 
Please make the punctuation consistent throughout the article. There are two different conventions updated in the article.

Reply:
Updated.